command/command.go

```diff
@@ -210,3 +211,33 @@ func setEnvVar(c *cli.Command) {
 		setEnvVar(&c.Subcommands[i])
 	}
 }
+
+// FingerprintFormatFlag returns a flag for configuring the fingerprint format.
+func FingerprintFormatFlag() cli.StringFlag {
+	return cli.StringFlag{
+		Name:  "format",
+		Usage: `The <format> of the fingerprint, it must be "hex", "base64", "base64-url", "base64-raw", or "emoji".`,
+	}
+}
+
+// GetFingerprintEncoding gets the fingerprint encoding from the format flag.
+func GetFingerprintEncoding(format, defaultFmt string) (fingerprint.Encoding, error) {
+	if format == "" {
+		format = defaultFmt
+	}
+
+	switch strings.ToLower(strings.TrimSpace(format)) {
+	case "hex", "":
+		return fingerprint.HexFingerprint, nil
+	case "base64":
+		return fingerprint.Base64Fingerprint, nil
+	case "base64url", "base64-url":
+		return fingerprint.Base64URLFingerprint, nil
+	case "base64raw", "base64-raw":
+		return fingerprint.Base64RawFingerprint, nil
+	case "emoji", "emojisum":
+		return fingerprint.EmojiFingerprint, nil
+	default:
+		return 0, errors.Errorf("error parsing fingerprint format: '%s' is not a valid fingerprint format", format)
+	}
+}
```

Comment:
base64.RawStdEncoding is missing, and is the default one for SSH fingerprints. See the [implementation].(https://github.com/golang/crypto/blob/ae814b36b87190c757eede9bc2d32ed77df88551/ssh/keys.go#L1479-L1483)

So `base64-raw` should be equivalent to base64.RawStdEncoding, and a new one of `base64-url-raw` should be base64.RawURLEncoding.

Reply:
Done.